rootfs/api/models/config.py

```diff
@@ -105,6 +112,40 @@ def set_tags(self, previous_config):
 
         raise DeisException(message)
 
+    def set_healthcheck(self, previous_config):
+        data = getattr(previous_config, 'healthcheck', {}).copy()
+        new_data = getattr(self, 'healthcheck', {}).copy()
+        # update the config data for healthcheck if they are not
+        # present for per proctype
+        if 'livenessProbe' in data.keys() or 'readinessProbe' in data.keys():
+            prev_data = data.copy()
+            data = {}
+            data['web/cmd'] = prev_data
+        if 'livenessProbe' in new_data.keys() or 'readinessProbe' in new_data.keys():  # noqa
+            lat_data = new_data.copy()
+            new_data = {}
+            new_data['web/cmd'] = lat_data
```

Comment:
this can be changed to a `new_data = {'web/cmd': new_data.copy()}` - Same with the above